doc/blog/Wasm-JS series/Blog - WASM.md

```diff
@@ -0,0 +1,672 @@
+
+  ``` csharp
+  // Setting only one CSS style
+  SetStyle("text-shadow", "2px 2px red");
+  
+  // Setting many CSS styles at once using C# tuples
+  SetStyle(("text-shadow", "2px 2px blue"), ("color", "var(--app-bg-color)"));
+  ```
+
+* The method `base.ResetStyle()` can be used to set CSS styles to their default values. Example:
+
+  ``` csharp
+  // Reset text-shadow style to its default value
+  ResetStyle("text-shadow");
+  
+  // Reset both text-shadow and color to their default values
+  ResetStyle("text-shadow", "color");
+  ```
+
+* The methods `base.SetAttribute()` and `base.RemoteAttribute()` can be used to set HTML Attributes on the html element. Example:
+
+  ``` csharp
+  // Set the "href" attribute of an <a> element
+  SetAttribute("href", "#section2");
+  
+  // Set many attributes at once
+  SetAttribute(("target", "_blank"), ("referrerpolicy", "no-referrer"));
+  
+  // Remove attribute from DOM element
+  RemoveAttribute("href");
+  ```
+
+* The method `base.SetHtmlContent()` could be used to set arbitrary HTML content as child of the control.
+
+  ``` csharp
+  SetHtmlContent("<h2>Welcome on Uno Platform!</h2>");
+  ```
+
+  > Note: don't use this unless your control doesn't include any _managed_ children, or you'll get unexpected results.
+
+* Finally, it's possible to invoke an arbitrary bunch of JavaScript code by using the _static_ method `WebAssembleRuntime.InvokeJS()`. Once there, it directly in the browser, so it's possible to make almost anything. It's possible to use the `HtmlId` property of the element to locate it in JavaScript code.
+  If the control has been loaded (after de `Loaded` router event), it will be available directly by calling the JavaScript `document.getElementById()`. But it's also possible to access it before using the  `Uno.UI.WindowManager.current.getView(<HtmlId>)` function in JavaScript.
+
+To illustrate how it's possible to use this in a real application, let's create one to integrate a pretty simple _syntax highlighter_ named [`PrismJS`](https://prismjs.com/).
+
+# Sample - Integration of PrismJS in a project
+
+## 0. Before starting
+
+📝 To reproduce the code in this article, you must [prepare your development environment](https://platform.uno/docs/articles/get-started.html).
+
+## 1. Creating the projects
+
+🎯 This section is very similar to the [_Getting Started_ in the official documentation](https://platform.uno/docs/articles/getting-started-tutorial-1.html).
+
+1. Start **VisualStudio 2019**
+2. Click `Create a new project`
+   ![image-20200325113112235](image-20200325113112235.png)
+3. **Search for "Uno"** and pick `Cross-Platform App (Uno Platform)`. Pay attention to select the `App`, not the `Library` project type.
+   ![image-20200325113532758](image-20200325113532758.png)
+   Select it and click `Next`.
+4. Give a project name and folder as you wish. It will be named `PrismJsDemo` here.
+5. Click `Create` button.
+6. Delete the `.Droid`, `.iOS` and `.UWP` projects.
```

Comment:
Is deleting the other heads a bit extreme? Maybe just note to not expect the other heads to compile without additional techniques (can link again to https://platform.uno/docs/articles/platform-specific-csharp.html)

Reply:
I was'nt surepon what is the best suggestion to make here. Maybe we should keep them to eventually make them multi-platform.